options.ts

```diff
@@ -868,6 +868,39 @@ const options = mkOptions(OPTIONS, {
             ignored: opt(''),
             show_numbered: opt(false),
             showWsIcons: opt(false),
+            showApplicationIcons: opt(false),
+            applicationIconOncePerWorkspace: opt(true),
+            applicationIconMap: opt<ApplicationIcons>({
+                '[dD]iscord': '󰙯',
+                '^thunderbird': '',
+                'class:wezterm$': '',
+                'draw.io': '󰇟',
+                'firefox-developer-edition': '',
+                'google-chrome': '',
+                'title:.*youtube': '',
+                Spotify: '󰓇',
+                chromium: '',
+                code: '󰨞',
+                dbeaver: '',
+                edge: '󰇩',
+                evince: '',
+                firefox: '',
+                foot: '',
+                keepassxc: '',
+                keymapp: '',
+                kitty: '',
+                obsidian: '󰠮',
+                password$: '',
+                qBittorrent$: '',
+                rofi: '',
+                slack: '',
+                spotube: '󰓇',
+                steam: '',
+                telegram: '',
+                vlc: '󰕼',
+            }),
```

Comment:
Prefer to keep a set of defaults in code somewhere. As this list grows and as we add more defaults, this will get unwieldy and annoying for the user to add to and modify.

We should move this to a constants file - maybe `/lib/constants/workspaces.ts` and then bring them into our workspace code. We can make it so user defined ones are concatenated to the beginning of our hard-coded defaults object so they take precedence.

This way the applicationIconMap can be empty by default.

Reply:
Done